### PR TITLE
Unskip aws test

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19836

## Description
unskip `AWS - CloudWatchLogs test playbook`
move `AWS - CloudTrail test playbook` to NonCircleTests

## Additional changes
Fix `print_color` to support  special characters

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
For `AWS - CloudWatchLogs test playbook` do you know why we don't have the proxy checkbox? If this could be developed on our side please open an issue add it as the reason in conf and remove from the unskipped section. If this is unsolvable please add it as a short explanation on conf, for example: `SDK does not support proxy option`

**Comment by ShahafBenYakir** [comment]
@adi88d Don't forget to add the explanation why no proxy checkbox could be added to this integrariton.
@avidan-H Can you have a look at this? The second @adi88d reverted the color changes his build passed. Maybe we did something wrong there?

**Comment by avidan-H** [comment]
@adi88d post the error you were getting here

**Comment by adi88d** [comment]
@avidan-H 

Traceback (most recent call last):
  File "./Tests/test_content.py", line 643, in <module>
    main()
  File "./Tests/test_content.py", line 628, in main
    execute_testing(server, ami_instance_ip, server_version, server_numeric_version)
  File "./Tests/test_content.py", line 581, in execute_testing
    proxy, is_ami)
  File "./Tests/test_content.py", line 103, in print_test_summary
    print_warning('\t - ' + playbook_id + ' - ' + reason)
  File "/home/circleci/project/Tests/test_utils.py", line 35, in print_warning
    print_color(warning_str, LOG_COLORS.YELLOW)
  File "/home/circleci/project/Tests/test_utils.py", line 27, in print_color
    print('{}{}{}'.format(color, obj, LOG_COLORS.NATIVE))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 50: ordinal not in range(128)

Exited with code exit status 1

File: Tests/test_utils.py [hunk 1]
```diff
@@ -23,8 +23,8 @@ class LOG_COLORS:
 
 
 # print srt in the given color
-def print_color(obj, color):
-    print('{}{}{}'.format(color, obj, LOG_COLORS.NATIVE))
+def print_color(str, color):
+    print(color + str + LOG_COLORS.NATIVE)
```
Comment (avidan-H) [inline Tests/test_utils.py]: try this
```suggestion
    print(u'{}{}{}'.format(color, obj, LOG_COLORS.NATIVE))
```

**Pull request merged**